src/api.js

```diff
@@ -55,6 +56,7 @@ export default class API extends mapboxgl.Control {
         console.log('MapboxGL Draw: Unsupported geometry type "' + feature.geometry.type + '"');
         return;
     }
+    feature.geojson.properties.permanent = permanent;
```

Comment:
I don't think we should be changing (or adding to) the user's data's metadata. Perhaps a feature can be marked permanent in a geometry class field rather than directly in the GeoJSON properties.

Basically, if you load a bunch of features into gl-draw for editing it shouldn't come out with a bunch of random extra properties.